README.md

```diff
@@ -106,6 +106,21 @@ To install the plugin in the LLDB system plugin directory, use the
 npm copy `node_modules/llnode/llnode.so` to
 `/usr/lib/lldb/plugins`.
 
+To use llnode with a core dump the core dump needs to be loaded into lldb along with the exact executable that created the core dump. The executable contains information that lldb and llnode need to make sense of the data in the core dump.
+
+To load the core dump when starting lldb use:
+```
+> lldb ~/work/node-sdks/node-v6.5.0-linux-x64/bin/node -c ~/work/core
```

Comment:
Maybe use fictitious paths here (e.g. `/path/to/bin/node` and `/path/to/core`) or clarify that the paths are example paths.

As well, I'd replace the `>` with a `$` to make it clear it's meant to be run from the shell.

Lastly, can you wrap lines at 80 columns?  Cheers.